neuralmonkey/runners/perplexity_runner.py

```diff
@@ -5,7 +5,7 @@
 computes its perplexities given the decoder.
 """
 
-# tests: lint
+# tests: lint, mypy
```

Comment:
To stejně znamená, že by se to dostalo do masteru až po `decoder_refactor`. Podle mne jsou dva přípustné postupy: buď zrušit tenhle request a nechat smazání komentářů na později, nebo trvat na smazání komentářů v tomhle PR a pak mít komplikovanější merge toho monster PR. Mne je to celkem jedno.